.pre-commit-config.yaml

```diff
@@ -3,7 +3,7 @@ repos:
     rev: v4.3.0
     hooks:
       - id: check-yaml
-        exclude: ^(helm/wrongsecrets-ctf-party/templates/|helm/test.tmp.yaml)
+        exclude: ^(helm/wrongsecrets-ctf-party/templates/|helm/test.tmp.yaml|azure/k8s/)
```

Comment:
Why do these need to be excluded?

Reply:
Because The pod-id files contain multiple configurations in the same file. Now they are .yaml.tpl but if the user generates them to .yaml. They need to be excluded.

Should I not cover this and leave it to users?